pkg/controller/controller.go

```diff
@@ -90,6 +90,7 @@ func NewControllerService(k8sClient *k8s.KubeClient, logger *logrus.Logger,
 // Probe is the implementation of CSI Spec Probe for IdentityServer.
 // This method checks if CSI driver is ready to serve requests
 // overrides same method from defaultIdentityServer struct
+// TODO need to introduce new liveness probe since node service doesn't discover AvailableCapacity anymore
```

Comment:
Is this really required?

Reply:
Currently liveness and readiness probes depend on the readiness of Node pods. They become ready if Discover function is successful. When Node discovers all drives, ac and volumes, Controller starts working. With proposed implementation, Controller starts working even if not all AC are created. If it's ok for CSI, it is't required.